src/SprykerMiddleware/Zed/Process/Business/Mapper/Mapper.php

```diff
@@ -124,21 +121,7 @@ protected function mapCallable(array $result, array $payload, string $key, calla
     protected function mapArray(array $result, array $payload, string $key, array $value): array
```

Comment:
Seems that mapKey and mapArray methods are similar now. May we need to leave only one if there aren't any difference

Reply:
fixed